paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -129,13 +133,15 @@ def get_kubernetes_resource_request(
 
 def get_kubernetes_metadata(
     pod: V1Pod,
-) -> Tuple[Optional[str], Optional[str], Optional[str]]:
+) -> Tuple[Optional[str], Optional[str], Optional[str], Optional[str], Optional[str]]:
     labels = pod.metadata.labels or {}
     node_selectors = pod.spec.node_selector or {}
+    pod_name = pod.metadata.name
```

Comment:
Would it be safer to use a get here ? I recon it's highly unlikely that a pod doesn't have an name but I'm not 100% sure a pod can not have an IP

Reply:
'get' won't work because pod metadata / status objects do not support this method. pod.metadata.name and pod.status.pod_ip are actually methods, in an unlikely situation pod doesn't have a name / IP assigned these methods just return 'None', which is going to be passed as 'null' to Splunk.